docs/learn-more/vscode/python-script.rst

```diff
@@ -222,86 +222,87 @@ Once you've created a Python file you need to select your interpreter. In short,
 
             .. tip::
 
-                In VS Code, pressing :kbd:`Ctrl+Shift+P` brings down a search bar where you can search for anything you need help with
+                In VS Code, pressing :kbd:`Ctrl+Shift+P` brings down a search bar where you can search for anything you need help with.
 
+            .. tip::
```

Comment:
doesn't it look weird with two tips, right next to each other...?

Reply:
Actually, this belongs to the next section that should be finished once Sotero finishes the video. I'll move it under the ``.. todo::`` and push.